middleware/passport/aadStrategy.ts

```diff
@@ -0,0 +1,79 @@
+import { IProviders } from '../../transitional';
```

Comment:
As a new file, I'd suggest dropping the header from the other files in here, too, to make the license terms clear.